pkgs/by-name/li/libtsm/package.nix

```
@@ -9,13 +9,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libtsm";
-  version = "4.0.2";
+  version = "69922bde02c7af83b4d48a414cc6036af7388626";
```

Please use the unstable format

```suggestion
  version = "4.0.2-unstable-2040-22-22";
```
and fix the date